doc/python_bindings.rst

```diff
@@ -4,23 +4,71 @@
 Using Drake from Python
 ***********************
 
-A limited subset of the Drake C++ functionality is available from Python. The Drake Python bindings are generated using `pybind11 <https://github.com/pybind/pybind11>`_, which means that every function or class which is exposed to C++ has been explicitly enumerated in one of the source files inside the ``drake/bindings/pybind11`` folder. These bindings are installed as a single package called ``pydrake``.
+A limited subset of the Drake C++ functionality is available from Python. The
+Drake Python bindings are generated using `pybind11
+<https://github.com/pybind/pybind11>`_, which means that every function or
+class which is exposed to C++ has been explicitly enumerated in one of the
+source files inside the ``drake/bindings/pybind11`` folder. These bindings are
+installed as a single package called ``pydrake``.
+
+Building the Python Bindings
+----------------------------
+
+To use the Python bindings from Drake externally, we recommend using CMake.
+As an example:
+
+.. code-block:: shell
+
+    git clone https://github.com/RobotLocomotion/drake.git
+    mkdir drake-build
+    cd drake-build
+    cmake ../drake
```

Comment:
probably just `..` now?

Reply:
It's actually an out-of-source build. Would you prefer that it be made in-source?

Reply:
ah, no.  not at all.  just thought it might have been a hold-over from before we moved the directories up.  disregard.